models/blog-series.ini

```diff
@@ -0,0 +1,9 @@
+[model]
+name = Blog Series
```

Comment:
Could you call this `blog-series-collection` or `blog-series-list` to make it clear that it's multiple series?